<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rental</class>
 <widget class="QMainWindow" name="Rental">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Прокат касет</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="rootLayout" stretch="1,1">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <layout class="QVBoxLayout" name="tapesLayout">
        <item>
         <widget class="QLabel" name="tapesLabel">
          <property name="text">
           <string>Таблиця &quot;Касети&quot;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tapesTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="tapesButtons">
          <item>
           <widget class="QPushButton" name="tapesAddButton">
            <property name="text">
             <string>Додати</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tapesSaveButton">
            <property name="text">
             <string>Зберегти</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tapesDeleteButton">
            <property name="text">
             <string>Видалити</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="customersLayout">
        <item>
         <widget class="QLabel" name="customersLabel">
          <property name="text">
           <string>Таблиця &quot;Клієнти&quot;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="customersTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="customersButtons">
          <item>
           <widget class="QPushButton" name="customersAddButton">
            <property name="text">
             <string>Додати</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="customersSaveButton">
            <property name="text">
             <string>Зберегти</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="customersDeleteButton">
            <property name="text">
             <string>Видалити</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout" stretch="7,3">
      <item>
       <layout class="QVBoxLayout" name="rentalsLayout">
        <item>
         <widget class="QLabel" name="rentalsLabel">
          <property name="text">
           <string>Таблиця &quot;Прокат&quot;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="rentalsTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="rentalsButtons">
          <item>
           <widget class="QPushButton" name="rentalsAddButton">
            <property name="text">
             <string>Додати</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rentalsSaveButton">
            <property name="text">
             <string>Зберегти</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rentalsDeleteButton">
            <property name="text">
             <string>Видалити</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="infoLayout">
        <item>
         <widget class="QLabel" name="tapeInfoLabel">
          <property name="text">
           <string>Інформація про касету</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="tapeInfoText"/>
        </item>
        <item>
         <widget class="QLabel" name="customerInfoLabel">
          <property name="text">
           <string>Інформація про клієнта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="customerInfoText"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
